pontoon/base/views.py

```diff
@@ -56,7 +56,15 @@
 
 def home(request):
     """Home view."""
+
+    user = request.user
     project = Project.objects.get(id=1)
+
+    # Redirect user to the selected home page or '/'. Because of circular redirects at home() this logic is moved to
+    # a separate view.
```

Comment:
You can remove the 2nd sentence now and make the comment shorter than 100 characters by doing that. :)